lib/mail_room/cli.rb

```diff
@@ -25,6 +25,10 @@ def initialize(args)
           options[:quiet] = true
         end
 
+        parser.on('--log-exit-as-json') do
```

Comment:
I was hoping for `--log-exit-as` with the value passed to be `"json"`. This would allow us to use the same argument with other formats in the future. E.g., `--log-exit-as=keyvalue` would allow us to output as `time=XXX message=YYYYYYYY` or any other format we need to support.

Reply:
Nice, all great ideas @tpitale! 💡 

I've pushed up a fully tested version with `--log-exit-as` as a flag. What do you think?